libs/sdk-ui-dashboard/src/presentation/dialogs/ExportDialogProvider.tsx

```diff
@@ -0,0 +1,18 @@
+// (C) 2021 GoodData Corporation
+import { ExportDialog } from "@gooddata/sdk-ui-kit";
+import React, { useCallback } from "react";
+import { useExportDialogContext } from "../dashboardContexts";
+
+/**
+ * @internal
+ */
+export const ExportDialogProvider: React.FC = () => {
+    const { closeDialog, dialogConfig, isOpen } = useExportDialogContext();
+
+    const onClose = useCallback(() => {
```

Comment:
Pls extract variable for dep. first,

const {onClose: dialogConfigOnClose} = dialogConfig;

than use it in dep. array

Reply:
updated